flow/Net2.actor.cpp

```diff
@@ -911,6 +932,7 @@ class SSLConnection final : public IConnection, ReferenceCounted<SSLConnection>
 			} else {
 				// Otherwise use flow network thread
 				BindPromise p("N2_AcceptHandshakeError"_audit, UID());
```

Comment:
Could you pass `self->id` here instead of `UID()`?
It's a useful way to track entire lifetime of a connection. I don't know why it isn't used here.